content/java/concepts/date/terms/from/from.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.from()' 
+
+Description: 'Returns an instance of a Date'
+
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+  - 'Discrete Math'
+
+Tags: 
+  - 'Date'
+  - 'Time'
+  - 'Methods'
+
+CatalogContent: 
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **'Date.from'** method returns an instance of a Date that is retrieved from an Instant object.  An Instant is more precise than a Date because it reports past milliseconds to nanoseconds.
```

Comment:
```suggestion
The **`Date.from()`** method returns an `Instant` object of a given `Date`. An `Instant` provides more accuracy than a `Date` because it reports nanoseconds (while `Dates` are limited to milliseconds).
```